collections/_infographics/energy/electricity-per-capita-world.md

```diff
@@ -14,4 +14,29 @@ data-orig:
   - [ "UN (population)", "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/1_Population/WPP2019_POP_F01_1_TOTAL_POPULATION_BOTH_SEXES.xlsx" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/elektrina-na-osobu-svet" %}
+## How to read this graph
+
+* The graph shows the **worldwide development of electricity produced per person by individual sources in the years 2000-2019**, as well as a comparison with selected world regions. Values are given in **<glossary id="w">kWh</glossary> of electricity produced per person per year**.
+* All graphs have the same vertical axis, so for regions with low average consumption per person, the filled part occupies only a minimal area of the graph.
+* However, the per capita comparison does not account for the different sizes of each region, so **the right column completes the regions' share of total global electricity generation in 2019**. The supplementary infographic, [Electricity generation in world regions](/infographics/electricity-per-capita-world), compares regions in terms of total electricity generation.
+
+## Comments on the graph
+
+* **Electricity production per person in the Czech Republic in 2019 was 8,066 kWh, significantly higher than the averages in the regions shown, except for the USA.** The Czech Republic is also a significant exporter of electricity, so the average electricity consumption per person was lower, at 6,891 kWh. Even this value is more than double the world average.
+* Globally, electricity production increased by 75% between 2000 and 2019, while at the same time, the population has grown by 1.6 billion people. n per capita terms, electricity generation increased by 39%, and in absolute terms, there was an increase in the capacity of all electricity generation sources except nuclear. The share of coal in electricity production fell slightly (from 37.9% to 35.2%), and the percentage of nuclear power fell even more significantly (from 16.9% to 10.5%). In contrast, there was an increase in the share of natural gas (from 18.2 % to 23.5%) and renewable sources (from 19.5% to 27.3% due to the development of wind and solar power plants).
```

Comment:
"the population" to "the global population"
"n per capita" to "In per capita"

debatable: I would change
"in the capacity of all electricity generation sources except nuclear" to "in electricity production by all electricity generation sources except in the nuclear sector" because we aren't talking about capacity, we are talking about actual production. But it's not a clear error.

Reply:
Fix the first two items (thanks for catching that typo where the "I" from "In" went missing).

Reply:
The original text states: "V přepočtu na osobu se výroba elektřiny zvýšila o 39 %, v absolutních hodnotách došlo k navýšení kapacit všech zdrojů výroby elektřiny s výjimkou jaderných."

Both deepl and google translate initially interpreted `kapacit` as `capacity` (alternatively `production capacity`). I will update but please let me know what you think. I won't resolve this yet

Reply:
@tjarnikova I updated the link at the top. Could you have another look this PR and specifically at this comment?

Reply:
@willemfrishert I looked at the original czech text and I see your point. For consistency, let's stick to the capacity terminology, ie, no change.